src/bot/handlers.py

```diff
@@ -46,14 +49,24 @@ async def start(update: Update, context: CallbackContext) -> None:
     await register(update, context)
 
 
-async def register(update: Update, context: CallbackContext) -> None:
+async def register(
+    update: Update,
+    context: CallbackContext,
+) -> None:
     """Инициализация формы регистрации."""
+    telegram_user_id = update._effective_user.id
+    session = get_session()
+    registration_service = await get_user_service_callback(session)
+    user = await registration_service.get_user_by_telegram_id(telegram_id=telegram_user_id)
+    query = None
+    if user:
+        query = urllib.parse.urlencode(user.__dict__)
     await update.message.reply_text(
         "Нажмите на кнопку ниже, чтобы перейти на форму регистрации.",
         reply_markup=ReplyKeyboardMarkup.from_button(
             KeyboardButton(
                 text="Зарегистрироваться в проекте",
-                web_app=WebAppInfo(url=settings.registration_template_url),
+                web_app=WebAppInfo(url=WEBAPP_URL.format(url=settings.registration_template_url, query=query)),

```

Comment:
Нет смысла выносить константу, поскольку переиспользовать её, вероятнее всего, не будем, а формат точно не изменится.
```suggestion
web_app=WebAppInfo(url=f"{settings.registration_template_url}?{query}"),
```

Reply:
Сделано